src/components/Map/index.jsx

```diff
@@ -160,8 +160,8 @@ Map.propTypes = {
 
 Map.defaultProps = {
   mapOptions: {
```

Comment:
Are we using this defaults? 
I think we are using those how lives at App.js and I think It could be a little mess if we keep two defaults. 
Can you check which one are the good ones and delete the others?